grails-app/controllers/org/pih/warehouse/UrlMappings.groovy

```diff
@@ -967,6 +967,11 @@ class UrlMappings {
             action = [POST: "submitCount"]
         }
 
+        "/api/facilities/$facility/cycle-counts/$cycleCountId/recount" {
```

Comment:
Let's pluralize the subresource to make it clear that it's a resource and not an action.